imx.mk

```diff
@@ -175,22 +176,66 @@ $(ROOT)/out/boot.scr: $(ROOT)/build/imx/u-boot_boot_script | $(ROOT)/out
 	$(U-BOOT_PATH)/tools/mkimage -T script -C none -n 'Boot script' \
 		-d $< $@
 
+################################################################################
+# Flash images
+################################################################################
+USE_PERSISTENT_ROOTFS ?= 0
+
+# Configuration of the BOOT partition
+FLASH_PARTITIONS_BLOCK_SIZE = 512
+FLASH_PARTITION_BOOT_START_BLOCK = 16384
+FLASH_PARTITION_BOOT_SIZE_IN_BYTES = \
+	$(shell echo $$(( 64 * 1024 * 1024 )))
+FLASH_PARTITION_BOOT_SIZE_IN_BLOCKS = \
+	$(shell echo $$(( ${FLASH_PARTITION_BOOT_SIZE_IN_BYTES} / ${FLASH_PARTITIONS_BLOCK_SIZE} )))
```

Comment:
Please use parentheses `()` for variables which are supposed to be expanded by make.

Reply:
Hey! Thanks for the review. I changed the brackets by the parentheses for the variables expanded by `make`. The code can be reviewed one more time.